modular_skyrat/code/modules/research/xenoarch/strange_rock.dm

```diff
@@ -0,0 +1,262 @@
+/obj/item/strangerock
+	icon = 'modular_skyrat/code/modules/research/xenoarch/fossil_and_artifact.dmi'
+	name = "strange rock"
+	desc = "This is a strange rock, it appears to have a relic encased."
+	icon_state = "strange"
+	item_state = "strange"
+
+	var/chosenitem = null
+	var/itemsafedepth = null
+	var/itembasedepth = null
+	var/itemactualdepth = null
+	var/dugdepth = null
+
+	var/tryagain = null
+
+/obj/item/strangerock/Initialize()
+	icon_state = pick("strange","strange0","strange1","strange2","strange3")
+	var/randomnumber = rand(1,200)
+	switch(randomnumber)
+		if(1 to 159)
+			chosenitem = pickweight(GLOB.bas_artifact)
+			itembasedepth = rand(30,40)
+			itemsafedepth = rand(3,6)
+			itemactualdepth = rand(itembasedepth - itemsafedepth,itembasedepth)
+		if(160 to 199)
+			chosenitem = pickweight(GLOB.adv_artifact)
+			itembasedepth = rand(40,60)
+			itemsafedepth = rand(6,12)
+			itemactualdepth = rand(itembasedepth - itemsafedepth,itembasedepth)
+		if(200)
```

Comment:
1 in 200 is reaaaallly rare, are you sure about this? That's generally rarer than getting a *super epic legendary drop* from shitty lootbox mobile games.

Reply:
Ill lower it to 100, we can see how bad it is.